PotreeConverter/src/main.cpp

```diff
@@ -179,7 +180,26 @@ PotreeArguments parseArguments(int argc, char **argv){
 
 		a.aabbValues = aabbValues;
 	}
-
+	if (args.has("metadata_processing")) {
+		vector<double> aabbValues;
+        auto vtmMetadata = parseVTMmetadata(a.metadataProcessingFile);
+		double paddingVTM = 300;
+		double paddingAltitude = 100;
+		double minEastingAdjusted   = vtmMetadata.minEasting - paddingVTM;
+		double minNorthingAdjusted  = vtmMetadata.minNorthing - paddingVTM;
+		double minAltitudeAdjusted  = vtmMetadata.minAltitude - paddingAltitude;
+		double maxEastingAdjusted   = vtmMetadata.maxEasting + paddingVTM;
+        double maxNorthingAdjusted  = vtmMetadata.maxNorthing + paddingVTM;
```

Comment:
Clean this indentation up. Match the indentation with other parts of the file. Sometimes it is tab vs. space.